ansible/files/zeppelin.sh

```diff
@@ -0,0 +1,200 @@
+          "name": "python"
+        }
+      ],
+      "dependencies": [],
+      "option": {
+        "remote": true,
+        "perNoteSession": false,
+        "perNoteProcess": false,
+        "isExistingProcess": false,
+        "port": "-1"
+      }
+    },
+    "2AJXGMUUJ": {
+      "id": "2AJXGMUUJ",
+      "name": "md",
+      "group": "md",
+      "properties": {
+        "zeppelin.interpreter.localRepo": "/usr/lib/zeppelin/local-repo/2AJXGMUUJ"
+      },
+      "interpreterGroup": [
+        {
+          "class": "org.apache.zeppelin.markdown.Markdown",
+          "name": "md"
+        }
+      ],
+      "dependencies": [],
+      "option": {
+        "remote": true,
+        "perNoteSession": false,
+        "perNoteProcess": false,
+        "isExistingProcess": false,
+        "port": "-1"
+      }
+    },
+    "2AKK3QQXU": {
+      "id": "2AKK3QQXU",
+      "name": "sh",
+      "group": "sh",
+      "properties": {
+        "shell.command.timeout.millisecs": "60000"
+      },
+      "interpreterGroup": [
+        {
+          "class": "org.apache.zeppelin.shell.ShellInterpreter",
+          "name": "sh"
+        }
+      ],
+      "dependencies": [],
+      "option": {
+        "remote": true,
+        "perNoteSession": false,
+        "perNoteProcess": false,
+        "isExistingProcess": false,
+        "port": "-1"
+      }
+    }
+  },
+  "interpreterBindings": {
+    "2A94M5J1Z": [
+      "2ANGGHHMQ",
+      "2AJXGMUUJ",
+      "2AM1YV5CU",
+      "2AKK3QQXU",
+      "2BRWU4WXC"
+    ],
+    "2C9CG1T4Z": [
+      "2ANGGHHMQ",
+      "2AJXGMUUJ",
+      "2AM1YV5CU",
+      "2AKK3QQXU",
+      "2BRWU4WXC"
+    ],
+    "2CAFFHNRH": [
+      "2ANGGHHMQ",
+      "2AJXGMUUJ",
+      "2AM1YV5CU",
+      "2AKK3QQXU",
+      "2BRWU4WXC"
+    ],
+    "2C9H41SEV": [
+      "2ANGGHHMQ",
+      "2AJXGMUUJ",
+      "2AM1YV5CU",
+      "2AKK3QQXU",
+      "2BRWU4WXC"
+    ]
+  },
+  "interpreterRepositories": [
+    {
+      "id": "central",
+      "type": "default",
+      "url": "http://repo1.maven.org/maven2/",
+      "releasePolicy": {
+        "enabled": true,
+        "updatePolicy": "daily",
+        "checksumPolicy": "warn"
+      },
+      "snapshotPolicy": {
+        "enabled": true,
+        "updatePolicy": "daily",
+        "checksumPolicy": "warn"
+      },
+      "mirroredRepositories": [],
+      "repositoryManager": false
+    },
+    {
+      "id": "local",
+      "type": "default",
+      "url": "file:///var/lib/zeppelin/.m2/repository",
+      "releasePolicy": {
+        "enabled": true,
+        "updatePolicy": "daily",
+        "checksumPolicy": "warn"
+      },
+      "snapshotPolicy": {
+        "enabled": true,
+        "updatePolicy": "daily",
+        "checksumPolicy": "warn"
+      },
+      "mirroredRepositories": [],
+      "repositoryManager": false
+    }
+  ]
+}
+EOF
+
+
+# Enable matplotlib support
+conda install -y pyqt=5
+sudo yum -y install libXdmcp xorg-x11-server-Xvfb
```

Comment:
We need X11 for the [interactive mode](http://matplotlib.org/faq/usage_faq.html#what-is-interactive-mode) which afaik will be supported in the next release.